assets/js/vendor/GoogleChart.js

```diff
@@ -179,7 +179,8 @@
     GoogleChart.prototype.bind = function(event, callback) {
       var _base;
 
-      (_base = this._listers)[event] || (_base[event] = this._addGoogleListener(event));
+      _base = this._listers;
+      _base[event] = this._addGoogleListener(event);
```

Comment:
It does not work when you change charts, otherwise.  Are the events local to the chart or global to the application?  I'm open to other ideas as to how to fix this.



> On Feb 1, 2015, at 2:27 PM, Tom Dooner <notifications@github.com> wrote:
> 
> In assets/js/vendor/GoogleChart.js:
> 
> > @@ -179,7 +179,8 @@
> >      GoogleChart.prototype.bind = function(event, callback) {
> >        var _base;
> >  
> > -      (_base = this._listers)[event] || (_base[event] = this._addGoogleListener(event));
> > +      _base = this._listers;
> > +      _base[event] = this._addGoogleListener(event);
> I'm skeptical of this - I think the left side of the || is to only have one googleListener per event, whereas this will add a new listener every time. I'm not sure the ramifications of that.
> 
> —
> Reply to this email directly or view it on GitHub.
>